Importer/Event/Error/InterpreterExceptionEvent.php

```diff
@@ -70,16 +70,16 @@ public function getIndex()
     /**
      * @return bool
      */
-    public function isStopped()
+    public function isAbort()
```

Comment:
I'm not sure. Isn't "is" a verb (in this context), and isn't "abort" also a verb? So it would be like "isDo" vs. "isDone".
But I don't mind either way.